pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go

```diff
@@ -48,6 +48,30 @@ type InternetCheckerSpec struct {
 	URLs []string `json:"urls,omitempty"`
 }
 
+type OperatorFlags map[string]string
+
+func (f OperatorFlags) GetBoolean(key string) (val bool, exists bool) {
+	v, ext := f[key]
+	if ext {
+		// Only accept a literal true, rather than accepting anything other than
+		// literal false as true
+		// TODO: Is this the best behaviour?
+		if v == "true" {
+			val = true
+		} else {
+			val = false
+		}
+	} else {
+		val = false
```

Comment:
Setting val = false at the start of the func, avoids the else here and above in the if ext, if v == "true" block

Reply:
In fact, it should default to false already